api/schema/types/PaymentType.js

```diff
@@ -7,7 +7,9 @@ module.exports = gql`
         amount: Int!
         date_incurred: String!
         date_paid: String
-        client_id: Int!
+        client_id: Int
+        contributor_id: Int
+        currency: String
```

Comment:
You made currency not nullable in the model so we should enforce it at the API level